### ISSUE TITLE
Ignore all SPDX files in repo (not just the tag-value flavour)

### ISSUE THREAD

**Issue by silverhook** [fsfe/reuse-tool#442]
Currently `reuse lint` ignores `*.spdx` extensions (i.e. tag-value SPDX format), which AFAICT is intentional.

I noticed though that it treats `*-spdx.rdf` (i.e. the RDF SPDX format) as a normal file and want it to have REUSE-compliant headers. I suspect the same will hold true for the JSON and YAML flavors as well.

As such, I suggest that `reuse lint` ignore all SPDX formats:
```
.spdx
.spdx.rdf
.spdx.json
.spdx.yaml
.spdx.yml
```

I suspect it falls under `_IGNORE_FILE_PATTERNS` in:
https://github.com/fsfe/reuse-tool/blob/4ce4c5fe2976536e9f2911048f1b6b308fa191ee/src/reuse/__init__.py#L48

See: https://spdx.github.io/spdx-spec/conformance/#44-standard-data-format-requirements

**Comment by mxmehl** [comment]
Makes absolutely sense, thanks for bringing this up.

I wonder whether these excludes shall also be mentioned in the spec? Workaround for https://reuse.software/spec/#copyright-and-licensing-information

**Comment by silverhook** [comment]
It would make sense to mention in the spec what all is excluded, I agree.

**Comment by mmurto** [comment]
Ran into an edge case with the change in #494. I maintain [a library](https://github.com/doubleopen-project/spdx-rs) for parsing SPDX documents. I started wondering why the the [REUSE check](https://github.com/doubleopen-project/spdx-rs/actions/workflows/reuse.yml) started failing without notable changes and finally noticed this change. We use some SPDX Documents as test data, for which we want to provide attribution with `dep5`. After `spdx.json` being excluded, REUSE no longer accepts a license for it in the `dep5`, which led to its licenes in the `LICENSES` directory now being an unused license.

Not sure if this should be handled by the tool/spec in some way.
